src/Microsoft.ML.DnnImageFeaturizer.ResNet101/ResNet101.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// This is an extension method to be used with the <see cref="DnnImageFeaturizerEstimator"/> in order to use a pretrained ResNet101 model.
+    /// The NuGet containing this extension is also guaranteed to include the binary model file. Note that when building the project
+    /// containing this extension method, the corresponding binary model will be downloaded from the CDN at
+    /// https://express-tlcresources.azureedge.net/image/ResNetPrepOnnx/ResNetPreprocess.onnx and
+    /// https://express-tlcresources.azureedge.net/image/ResNet101Onnx/ResNet101.onnx  and placed into the local app directory
+    /// folder under mlnet-resources.
+    /// </summary>
+    public static class ResNet101Extension
+    {
+        /// <summary>
+        /// If including this through a NuGet, the location of the model will be the same as of this file. This looks for the model there.
+        /// This should be the default way to use ResNet101 if importing the model from a NuGet.
+        /// </summary>
+        public static EstimatorChain<OnnxTransform> ResNet101(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string input, string output)
```

Comment:
`input` and `output` what?  Column names?

Reply:
Renamed for more clarity.

---
In reply to: [232776196](https://github.com/dotnet/machinelearning/pull/1447#discussion_r232776196) [](ancestors = 232776196)